openpgp/keys.go

```diff
@@ -44,6 +45,13 @@ type Identity struct {
 	Signatures    []*packet.Signature // all (potentially unverified) self-signatures, revocations, and third-party signatures
 }
 
+type UserAttribute struct {
```

Comment:
It's a bit confusing to name this struct the same as its first field. Perhaps we can name this one simply `Attribute`?

Reply:
the name is same with  [User Attribute](https://datatracker.ietf.org/doc/html/rfc4880#section-5.12)